internal/pkg/pkg.go

```diff
@@ -51,26 +64,24 @@ func New(path string) (*Pkg, error) {
 	if err != nil {
 		return nil, err
 	}
-	var relPath string
 	var absPath string
 	if filepath.IsAbs(path) {
-		// If the provided path is absolute, we find the relative path by
-		// comparing it to the current working directory.
-		relPath, err = filepath.Rel(cwd, path)
-		if err != nil {
-			return nil, err
-		}
 		absPath = filepath.Clean(path)
 	} else {
 		// If the provided path is relative, we find the absolute path by
-		// combining the current working directory with the relative path.
-		relPath = filepath.Clean(path)
+		// combining the current working directory with the path.
 		absPath = filepath.Join(cwd, path)
 	}
-	return &Pkg{
-		UniquePath:  types.UniquePath(absPath),
-		DisplayPath: types.DisplayPath(relPath),
-	}, nil
+	dp, err := types.NewDisplayPath(filepath.Base(absPath))
```

Comment:
So you're only using the last element of the path? How does this align with the docs for NewDisplayPath?

Reply:
Removed NewDisplayPath method to avoid confusion. By default the display path is the package name. But if the package is a subpackage of the package on which the command is invoked, the DisplayPath is adjusted accordingly in `adjustDisplayPathForSubpkg` method.